### PR TITLE
Login, Signup, Home, React-navigation

### PR DESCRIPTION
**Pull request by raymondng1**
This was a pain to set up react navigation for transitions: 
1. npm install @react-navigation/native
2. npm install @react-navigation/stack
2. npm install react-native-gesture-handler react-native-reanimated react-native-screens react-native-safe-area-context @react-native-community/masked-view 

https://reactnavigation.org/docs/getting-started

Afterwards you may have to go to this ios folder and re-run **pod install** 

After installing all these dependencies:
1. npm run ios
2. Default page will be the login page
3. Clicking the Login and which will go to the home component(will need to add authentication and error handling) 
4. When you are at home, clicking the "Climbing Routes" will take all the Climbing Routes component
5. Can go back, click sign up and will go to the sign up component

_Next steps will be to add authentication and error handling for login and sign up_

### CODE REVIEW

File: ios/climbARmobile.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nries1) [inline ios/climbARmobile.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: What's this for?

File: Login.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          placeholderTextColor="#e4572e"
+        />
+        <TextInput
+          style={styles.inputBox}
+          placeholder="Password"
+          secureTextEntry={true}
+          placeholderTextColor="#e4572e"
+        />
+        <TouchableOpacity style={styles.button}>
+          <Text style={styles.buttonText}> Login </Text>
```
Comment (nries1) [inline Login.js]: The button doesn't do anything yet right?

**Comment by nries1** [comment]
Thanks @raymondng1! This looks like it was really complicated. I'm going to wait to approve this until the redux branch is merged. One question: I see you invoked an expo command in the description of the PR. Does that require us to use expo for this app? I thought you didn't set that up and instead used the react native CLI.

File: Login.js (hunk 2 above)
Comment (raymondng1) [reply Login.js]: the button should take you to the home screen after clicking it

File: ios/climbARmobile.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (raymondng1) [reply ios/climbARmobile.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: this is something that xcode spits out after running a build, will probably need to delete/ .gitignore it

File: Login.js (hunk 2 above)
Comment (raymondng1) [reply Login.js]: @nries1 oh nevermind I see that the new commits didn't get pushed through

**Comment by raymondng1** [comment]
Yeah you're right instead of the command expo, it should be "npm install"

File: Login.js (hunk 2 above)
Comment (raymondng1) [reply Login.js]: oh nevermind it did get pushed through, think you just might looking at an old branch

**Comment by nries1** [comment]
Nice job with this!

**Pull request merged**